README.md

```diff
@@ -109,7 +109,7 @@ account to create an application repository.
 
 Follow the steps below:
 
-1. Run `make prepare-olm version=0.12.0`. Bear in mind that if there's different versions
+1. Run `make prepare-olm version=0.13.0`. Bear in mind that if there's different versions
 in the `deploy/olm-catalog/kogito-operator/kogito-operator.package.yaml` file, every CSV must
 be included in the output folder. At this time, the script did not copy previous CSV versions to the
 output folder, so it must be copied manually.
```

Comment:
Meybe we can remove this from README, as now all the manifests will be present after running the `make prepare-olm`?

Reply:
+1

Reply:
I'll change once GH gets back.

Reply:
Actually this is needed to verify the CSV in a local cluster. I'll keep it.

Reply:
Yes, the command should be there but what I meant to be remove is this:
 ```
 every CSV must
be included in the output folder. At this time, the script did not copy previous CSV versions to the
output folder, so it must be copied manually.
```
As after running this we won't need to copy anything manually.